src/client/src/rt-components/open-fin/OpenFinChrome.tsx

```diff
@@ -0,0 +1,159 @@
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { css } from 'emotion'
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import { rules } from 'rt-styleguide'
+import { styled, StyledComponent } from 'rt-theme'
+
+export interface ControlProps {
+  minimize?: () => void
+  maximize?: () => void
+  close: () => void
+}
+
+export interface HeaderProps extends ControlProps {
+  hide?: boolean
+}
+
+export const OpenFinChrome = ({ children }) => (
+  <Root>
+    <Helmet>
```

Comment:
No, because it breaks the browser environment, and it's not a concern outside of OpenFin.

I suspect a lack of familiarity with `react-helmet` is playing a role here. It's widely used for most consumer facing react apps — just check the usage on NPM. I've used it for the last three years and haven't seen a better solution yet. 

It's also well maintained to boot.